perun-core/src/main/java/cz/metacentrum/perun/core/api/FacilitiesManager.java

```diff
@@ -59,6 +59,17 @@
 	 */
 	Facility getFacilityByName(PerunSession perunSession, String name) throws FacilityNotExistsException, PrivilegeException;
 
+	/**
+	 * Searches for the Facilities with specified ids.
+	 *
+	 * @param perunSession
+	 * @param ids
+	 * @return list of Facilities with specified ids
+	 * @throws InternalErrorException
```

Comment:
I don't think there is any reason to include `InternalErrorException` to javadoc, could you please delete it from the javadocs you added?

Reply:
Done.